### DISCUSSION TITLE
automations Integromat

### DISCUSSION THREAD

**Discussion by juliansworkshop** [Budibase/budibase#2649] [Help]
Regardless of how I structure the data or if I don't pass anything along I see a bunch of encrypted packets to and from where there supposed to go and a 400 status in the console.

bbapps                | Automation error - integromat - Error: Request failed with status code 400

**Comment by shogunpurple** [comment]
Hi @juliansworkshop - could I some screenshots of the configuration for your automation? I should be able to help out.
